### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by dikshasaini11**
Find the changes done to the readme file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,24 +26,36 @@ Below are the rules of the game under which we will design an application.
...
+1.bin: This folder has executable files and data files.
 
 2.build: This folder has all the object files generated using make file. 
 
 3.doc: This folder contains documents with instructions for users and developers.
 
-4. Include: This folder contains header files used in application
+4. Include: This folder contains header files used in application.
 
 5. lib:
```
Comment (sayali20773) [inline README.md:37]: Even though we are not using any library file, I think we should provide a description here.

Comment (dikshasaini11) [reply README.md:37]: Okay. will add that.

File: README.md [hunk 2]
```diff
@@ -1,23 +1,23 @@
...
 - Rupneet Kaur Gill Rupneetkaurgill@cmail.carleton.ca
+- Vidushi Gupta Vidushigupta@cmail.carleton.ca
 
 # Brief Description
 
 Below are the rules of the game under which we will design an application.
 1. Snakes and ladders game is a grid of 10 x10 squares. The entire grid is numbered from 1 to 100.
-2. *It is a multi-player game. 2 or more players are required.*
+2. *It is a multi-player game. 2 players are required.*
 3. Each player takes turns to roll a six-sided dice. The player moves as many squares forward as     number on the dice. A player gets a consecutive turn if player gets a six.
```
Comment (Vidushi187) [inline README.md:21]: Each player takes turns to roll a six-sided dice. The player moves as many squares forward as number on the dice. A player gets a consecutive turn if player gets a six.

File: README.md [hunk 3]
```diff
@@ -1,23 +1,23 @@
...
 4. Each player starts at 1. Each player needs to roll 1 or a 6 to start playing the game.
 5. If a player’s counter lands at bottom of ladder, the player moves to top of ladder.
```
Comment (Rupneetkaurgill) [inline README.md:23]: Similarly, if players counter lands on the top of the snake, the player moves to the bottom of the snake.

Comment (dikshasaini11) [reply README.md:23]: yes

**Pull request merged**